src/components/Header/Header.jsx

```diff
@@ -15,6 +15,12 @@ export const navLinks = [
     id: 'PROPOSITION_2',
     label: 'Proposition II'
   },
+  {
+    aria: 'Section 2 Hypotheses',
+    href: '/II.Hypotheses',
+    id: '2Hypotheses',
+    label: 'Section 2 Hypotheses'
+  },
```

Comment:
I also wonder if we should show slightly different nav items on the main header vs the slide down nav menu. The main head is going to get crowded.